File: JAVA_SIS104/app/src/main/java/com/example/prueba_java/MainActivity.java
package com.example.prueba_java;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    Button buttonCalculadora, buttGrafico, buttGrafico2, buttGrafico3, buttBD, buttService, buttServiceLocal;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        buttonCalculadora = findViewById(R.id.buttonCalculadora);
        buttGrafico = findViewById(R.id.buttonGrafico);
        buttGrafico2 = findViewById(R.id.buttonGrafico2);
        buttGrafico3 = findViewById(R.id.buttonGrafico3);
        buttBD = findViewById(R.id.buttonBD);
        buttService = findViewById(R.id.buttonService);
        buttServiceLocal = findViewById(R.id.buttonServiceLocal);

        buttonCalculadora.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, CalculadoraActivity.class);
                startActivity(intent);
            }
        });

        buttGrafico.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, GraficoActivity.class);
                startActivity(intent);
            }
        });

        buttGrafico2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, Grafico2Activity.class);
                startActivity(intent);
            }
        });

        buttGrafico3.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, ConsolaActivity.class);
                startActivity(intent);
            }
        });

        buttBD.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, BaseDatosActivity.class);
                startActivity(intent);
            }
        });

        buttService.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, WebServiceAlbumsActivity.class);
                startActivity(intent);
            }
        });

        buttServiceLocal.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, WebLocalActivity.class);
                startActivity(intent);
            }
        });
    }


}

File: KOTLIN_SIS104/app/src/main/java/com/example/prueba_kotlin/MainActivity.kt
package com.example.prueba_kotlin

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import com.example.prueba_kotlin.backend.BaseDatos
import com.example.prueba_kotlin.backend.Lugares
import com.example.prueba_kotlin.backend.operaciones

class MainActivity : AppCompatActivity() {



    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val buttoncalculadora = findViewById<Button>(R.id.buttonCalculadora)
        val buttonsalir = findViewById<Button>(R.id.buttonSalir)
        val buttoncrud = findViewById<Button>(R.id.buttonCrud)
        val buttongrafico = findViewById<Button>(R.id.buttonGrafico)
        val buttonparabola = findViewById<Button>(R.id.buttonParabola)
        val buttoncaratula = findViewById<Button>(R.id.buttonCaratula)
        val buttonsw = findViewById<Button>(R.id.buttonServiceWeb)

        buttonsalir.setOnClickListener{
            finish()
        }

        buttoncalculadora.setOnClickListener{
            val intent = Intent(this@MainActivity,CalculadoraActivity::class.java)
            startActivity(intent)
        }

        buttongrafico.setOnClickListener{
            val intent1 = Intent(this@MainActivity,GraficoActivity::class.java)
            startActivity(intent1)
        }

        buttoncrud.setOnClickListener{
            val intent2 = Intent(this@MainActivity,CrudActivity::class.java)
            startActivity(intent2)
        }

        buttonparabola.setOnClickListener {
            val intent3 = Intent(this@MainActivity,Practica_Parabola_Activity::class.java)
            startActivity(intent3)
        }

        buttoncaratula.setOnClickListener {
            val intent4 = Intent(this@MainActivity,CaratulaActivity::class.java)
            startActivity(intent4)
        }

        buttonsw.setOnClickListener {
            val intent5 = Intent(this@MainActivity,WS_Local_Activity::class.java)
            startActivity(intent5)
        }
    }
}